expo/tests/contexts/volunteer_context.py

```diff
@@ -0,0 +1,44 @@
+from tests.factories.gbe_factories import (
+    GenericEventFactory,
+    ProfileFactory,
+    ShowFactory,
+    VolunteerFactory,
+)
+from tests.factories.scheduler_factories import (
+    EventContainerFactory,
+    ResourceAllocationFactory,
+    SchedEventFactory,
+    WorkerFactory,
+)
+
+
+class VolunteerContext():
+    def __init__(self,
+                 profile=None,
```

Comment:
It doesn't look like "profile" is used anywhere in here?
Is there a way to reconcile bid.profile and profile if both arguments are passed in?  If not, maybe the contexts should be separated somehow?